packages/runtime/test/lib/testUtils.ts

```diff
@@ -160,12 +160,13 @@ export async function syncUntilHasEvent<TEvent extends Event>(
     return event;
 }
 
-export async function uploadOwnToken(transportServices: TransportServices, forIdentity?: string): Promise<TokenDTO> {
+export async function uploadOwnToken(transportServices: TransportServices, forIdentity?: string, password?: string): Promise<TokenDTO> {
```

Comment:
when just calling this password, you completely loose the possibility to properly test this for example with PINs. Why not allow to pass an object with password and passwordIsPin? Would also make the usage below cleaner.